pkg/vdbgen/vdb.go

```diff
@@ -419,6 +512,37 @@ func (d *DBGenerator) queryLocalPath(ctx context.Context, usage string) (string,
 	return commonPrefix, nil
 }
 
+// queryLocalDataSize will find data/depot size. It will pick the max among all nodes
+func (d *DBGenerator) queryLocalDataSize(ctx context.Context, usage string) (int64, error) {
+	var qtype QueryType
+	if usage == "DEPOT" {
+		qtype = DepotSizeQueryKey
+	} else if usage == "DATA,TEMP" {
+		qtype = CatalogSizeQueryKey
```

Comment:
Rather than have a mapping from `string` to key, I suggest have the caller pass in the key to use, and use that directly.